app.rb

```diff
@@ -6,6 +6,22 @@
 end
 require_relative 'lib/formatter'
 
+helpers do
+  # parse a text area.  We want an array of IDs coming out.
+  # Do the right thing for newline or comma separated values.
+  def parse(t)
+    t.split(/[,\n]/).map(&:strip).select{ |x| !x.empty? }
```

Comment:
You could use `reject(&:empty?)` here, instead of `select`.

Also, would you please update the local variable to something like `text_area` instead of `t`-- Rubyists tend to be verbose/explicit in variable naming :dash: